server/test/christmas-trees-admin-controller.es6

```diff
@@ -1,144 +1,204 @@
+/* eslint-disable no-unused-expressions */
+const chai = require('chai');
```

Comment:
tests copied from a separate WIP branch with revamped tests. Comments left in to facilitate future merging.